packages/sync/legacy/class.jetpack-sync-modules.php

```diff
@@ -18,7 +18,7 @@ class Jetpack_Sync_Modules {
 		'Jetpack_Sync_Module_Users',
 		'Jetpack_Sync_Module_Posts',
 		'Jetpack_Sync_Module_Import',
-		'Jetpack_Sync_Module_Protect',
+		'Automattic\\Jetpack\\Sync\\Modules\\Protect',
```

Comment:
Probably ought to be namespaced inside a package for the Protect module itself, like

`\Automattic\Jetpack\Protect\Sync_Module`

Then the protect Sync module should be registered when the Protect module is active - right?

Reply:
I'm going to handle that in an other PR